controllers/articles.controllers.js

```diff
@@ -0,0 +1,13 @@
+const { findArticle } = require("../models/articles.models")
+
+
+exports.getArticles = (req, res, next) => {

```

Comment:
This controller should be called something like 'getArticleById' as it's just dealing with getting a single article